README.md

```diff
@@ -118,6 +118,35 @@ Available files, and their formats:
      * One line per input read with taxonomic classification and detailed hit
        information.
 
+* `samplereads/`: Itermediate, example reads by taxonomic category
+  * ex: `SRR14530724.sr.tsv.gz`
+  * TSV
+  * 100,000 examples (or as many as available) from each sample of:
+    * a: all reads
+    * b: bacterial reads
+    * v: viral reads
+    * h: human viral reads
+  * Columns are:
+    * Category (a/b/v/h)
+    * Read ID
+
+* `readlengths`: Output, distribution of read lengths by taxonomic category
```

Comment:
Interesting addition! Does it make sense to also output the overall read length distribution across all taxonomic categories?

Reply:
It does make sense to do that, and it does do that ;) That is category "a" for "all"